client/src/components/Container.jsx

```diff
@@ -33,13 +35,27 @@ class Container extends React.Component {
     });
   }
 
+  getData() {
+    var itemId = Math.floor(Math.random() * 100);
+    axios.get(`/items/${itemId}`)
+      .then((res) => {
+        var storeId = res.data[0].store_id;
+        this.setState({
```

Comment:
Keep in mind evert time you call `setState` it triggers re-rendering which slows down the app.  When you call `this.getData();`, you're calling a cascade of 3 set states.  Not a huge deal, mostly a comment about efficiency/speed

Reply:
yep this is why I have the slowness, I just haven't had the time yet to optimize and figure out how to redo this to only set the state once

Reply:
maybe i could just set the data to individual arrays and then once all of the calls have completed i can set all the states to the corresponding arrays